Assignment1/ip.rb

```diff
@@ -0,0 +1,21 @@
+puts "Enter IP address "       #Print message.
+IP_add = gets.chomp            #Input from user.
+split = IP_add.split('.',0)    #Split input on .
+cls = split[0].to_i            #convert 0 index of splited array to integer
+if (1..127) === cls            #Checking between classes 
```

Comment:
Your approach is also correct, but you can use case statements like this:
`case cls
when 1..127
  puts "Class A"
when 128..191
  puts "Class B"
when 192..223
  puts "Class C"
when 224..239
  puts "Class D"
when 240..254
  puts "Class E"
else
  # Handle other cases if needed
end`